build/cqs.tsv

```diff
@@ -27,6 +26,7 @@ com.googlecode.efficient-java-matrix-library:core	0.26	compile
 com.google.code.gson:gson	2.8.1	compile
 com.googlecode.json-simple:json-simple	1.1.1	compile
 com.googlecode.juniversalchardet:juniversalchardet	1.0.3	compile
+com.google.flatbuffers:flatbuffers-java	1.9.0	compile
```

Comment:
it seems like we can just use an existing cq without creating a piggy-back... there is one here already: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18345